packages/rpc/src/client.ts

```diff
@@ -27,46 +33,68 @@ import type {
 	SetUserVoiceSettingsData,
 	RelationShip,
 } from './typings/structs';
-import { RPCCommands, RPCEvents, type LobbyType, RelationshipType } from './typings/types';
-import { pid as getPid, uuid } from './util';
-
-function subKey(event: string, args?: unknown[]) {
-	return `${event}${JSON.stringify(args)}`;
-}
+// eslint-disable-next-line import/extensions
+import { RPCCommands, RPCEvents, type LobbyType, RelationshipType, RPCErrorCodes } from './typings/types';
+// eslint-disable-next-line import/extensions
+import { pid as getPid } from './util';
```

Comment:
Can't comment on deleted lines 🙄 
```suggestion
import { RPCCommands, RPCEvents, type LobbyType, RelationshipType, RPCErrorCodes } from './typings/types.js';
import { pid as getPid } from './util.js';
```